src/Php81/Resources/stubs/CURLStringFile.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+// https://php.watch/versions/8.1/CURLStringFile#CURLStringFile-polyfill
+class CURLStringFile extends CURLFile
+{
+    public function __construct(string $data, string $postname, string $mime = "application/octet-stream")
```

Comment:
```suggestion
    public function __construct(string $data, string $postname, string $mime = 'application/octet-stream')
```

Reply:
Thank you, I changed it like you suggested.